bep32v01/Createfile.py

```diff
@@ -0,0 +1,137 @@
+import json
+import os
+from BidsFilestructure import FileStructure
+
+
+class CreatFile:
```

Comment:
same, this is an action name... weird for a class...

(and a E letter is missing.... CreatE)